atramhasis/alembic/versions/cb568ec81000_add_infer_concept_relations_column.py

```diff
@@ -0,0 +1,23 @@
+"""Add infer_concept_relations column.
+
+Revision ID: cb568ec81000
+Revises: 184f1bbcb916
+Create Date: 2020-02-19 09:29:17.619082
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = 'cb568ec81000'
+down_revision = '184f1bbcb916'
+
+
+def upgrade():
+    op.add_column(
+        'concept',
+        sa.Column('infer_concept_relations', sa.Boolean(), nullable=True)
```

Comment:
Should not be nullable and default value should be true. Still need to take a business decision if Atramhasis will actually support False, even though it makes sense for certain skosproviders.

Reply:
But the collection Table is the same as the Concept table. 
And because Concepts don't have this column it must be nullable.